.github/workflows/build-and-push-assets.yml

```diff
@@ -48,12 +48,17 @@ on:
       GITHUB_USER_NAME:
         description: Github Username used in "git config".
         required: true
+      GITHUB_USER_SSH_KEY:
+        description: Private SSH key associated to the GitHub user in Github Username.
```

Comment:
```suggestion
        description: Private SSH key associated with the GitHub user passed as `GITHUB_USER_NAME`.
```

Reply:
Done in [3855f95](https://github.com/inpsyde/reusable-workflows/pull/12/commits/3855f95897238b6b52c751cc2b284ef522b79e9c)